videoprocess/Makefile.am

```diff
@@ -37,6 +37,21 @@ TEST_LIBS = \
 vavpp_SOURCES = vavpp.cpp
 vavpp_LDADD   = $(TEST_LIBS)
 
+vppscaling_csc_SOURCES = vppscaling_csc.cpp
+vppscaling_csc_LDADD = $(TEST_LIBS)
+
+vppdenoise_SOURCES = vppdenoise.cpp
+vppdenoise_LDADD   = $(TEST_LIBS)
+
+vppsharpness_SOURCES = vppsharpness.cpp
+vppsharpness_LDADD   = $(TEST_LIBS)
+
+vppchromasitting_SOURCES = vppchromasitting.cpp
+vppchromasitting_LDADD   = $(TEST_LIBS)
+
+vppscaling_n_out_usrptr_SOURCES = vppscaling_n_out_usrptr.cpp
```

Comment:
Is this ready for check-in? Does this depend on libva related to change?

Reply:
this PR not depend on libva changes

Reply:
Here, you include vppscaling_n_out_usrptr.cpp, but I do not see vppscaling_n_out_usrptr.cpp is uploaded. Does vppscaling_n_out_usrptr.cpp depend on libva changes?